web_ui/src/pages/annotator/zoom/use-zoom-into-annotations.hook.ts

```diff
@@ -54,23 +50,19 @@ export const useZoomIntoAnnotation = (): void => {
         // We need to compare the image target (which is the initial target) with the current zoomTarget
         const imageTargetConfig = { x: 0, y: 0, width, height } as ZoomTarget;
 
-        const targetChanged = !isEqual(imageTargetConfig, zoomTarget);
-
-        if (targetChanged) {
-            setZoomTarget(imageTargetConfig);
-        }
+        return imageTargetConfig;
```

Comment:
feels weird that "resetZoom" returns anything. If we really need this shall we rename this to something like `originalZoom`?